packages/server/migrations/js/202001221523000-unlogged.js

```diff
@@ -0,0 +1,45 @@
+module.exports = {
+  up: async function (queryInterface) {
+    try {
+      await queryInterface.sequelize.query('BEGIN');
+
+      const tablesWithOrder = [
+        'Votes', 'Mutexes', 'FeedItems',
+        'NotificationStatuses', 'Notifications',
+      ];
+      for (const table of tablesWithOrder) {
+        await queryInterface.sequelize.query(
+          `ALTER TABLE "${table}" SET UNLOGGED;`,
```

Comment:
What does this do?

Reply:
This optimizes tables inside of Postresql. It will work a bit faster. "UNLOGGED" means that Postresql will not write logs for this table in the recovery log.